docs/user-guide/configuring-zowe-via-jcl.md

```diff
@@ -1,6 +1,129 @@
+:::note
+Configuring Zowe with JCL is currently in *technical preview*. In a later release, this will become the default method of configuration.
+:::
+
+Zowe can be configured on your system with JCL using the `zwe` commands, or by customizing and running JCL entirely manually. Both of these configuration methods require a [`zowe.yaml` configuration file](./installandconfig.md#zowe-configuration-file-zoweyaml). Between these two options, we recommend sticking with `zwe` commands rather than manually submitting JCL from scratch, as there's additional validation we can perform on your configuration prior to creating and submitting JCL.
+
+- [Getting started with `zwe` and JCL](#getting-started-with-zwe-and-jcl)
+  - [Generating JCL](#generating-jcl)
+  - [Adding Job Parameters to generated JCL](#adding-job-parameters-to-generated-jcl)
+  - [Reviewing JCL before submission](#reviewing-jcl-before-submission)
+  - [Following existing `zwe` command documentation](#following-existing-zwe-command-documentation)
+  - [Following existing z/OSMF workflows](#following-existing-zosmf-workflow-documentation)
+- [Getting started with manual JCL submission](#getting-started-with-manual-jcl-submission)
+  - [Preparing the JCL](#preparing-the-jcl)
+  - [Manual JCL Core Tasks](#manual-jcl-core-tasks)
+  - [Manual JCL Keyring Tasks](#manual-jcl-keyring-tasks)
+  - [Manual JCL Caching Service VSAM Task (Deprecated)](#manual-jcl-caching-service-vsam-task-deprecated)
+
+## Getting started with `zwe` and JCL
+
+By default, `zwe` will not submit solely JCL to configure Zowe, instead relying on a mix of unix services and JCL. To enable `zwe` to rely **solely** on JCL, you can either add `--jcl` to the end of every `zwe install` or `zwe init` command, or set `zowe.setup.jcl.enable` in your zowe.yaml file. 
+
+Example zowe.yaml:
+```yaml
+zowe:
+  setup:
+    jcl:
+      enable: true
+```
+
+Example commands: `--jcl` overrides `zowe.setup.jcl.enable`
+```shell
+zwe install -c /path/to/your/zowe.yaml --jcl
+zwe init mvs -c /path/to/your/zowe.yaml --jcl
+```
+
+### Generating JCL
+
+Running `zwe init` commands with JCL require that you to first generate JCL using values present in your `zowe.yaml` file. To do this, run:
+
+```shell
+zwe init generate -c /path/to/your/zowe.yaml
+```
+
+This takes the configuration values present in your `zowe.yaml` file, uses them to populate JCL templates in `SZWESAMP`, and creates a `JCLLIB` dataset with the final generated JCL. The `JCLLIB` will be created using the value of `zowe.setup.dataset.jcllib`. 
+
+For example, `zwe init generate` with the below zowe.yaml creates the dataset `MY.DS.PREFIX.JCLLIB`. If this dataset already exists, you must add `--allow-overwrite` to the `init generate` command.
```

Comment:
>  If this dataset already exists, you must add `--allow-overwrite` to the `init generate` command.

Nope, this dataset is always deleted and created by [`ZWEGEN00`](https://github.com/zowe/zowe-install-packaging/blob/27796ca23660798392beae2d71c421a34566f4b1/files/SZWEEXEC/ZWEGEN00#L136-L138).